lib/constraints/is_admin.rb

```diff
@@ -0,0 +1,27 @@
+class Constraints::IsAdmin
```

Comment:
What about to call it just `Constraints::Admin`? `is_` or `Is` prefix seems to "Javaish" to me (like `get_` prefix for getters).

Reply:
done!